File: C++/512.cpp
#include <iostream>

using namespace std;

// 512 - Döner Sospechoso - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for (int i = 0; i<loop; i++) {
        int kgRabbit = 0;
        int kgHorse = 0;
        cin >> kgRabbit;
        cin >> kgHorse;
        cout << (kgRabbit * 100) / (kgRabbit + kgHorse) << endl;
    }
}

File: Java/512.java
import java.util.Scanner;

/**
 *
 * 512 - Döner Sospechoso - Iván - https://github.com/wildfireOfMine
 */
public class main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        for (int i = 1; i <= a; i++) {
            int b = sc.nextInt();
            int c = sc.nextInt();
            System.out.println((b * 100) / (b + c));
        }
    }
}
